docs/blog/2021-01-19-frontend-asset-passthrough.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Frontend Asset Passthrough
+authors:
+  - name: Mike Cutalo
+    url: https://github.com/mcutalo88
+    avatar: https://avatars1.githubusercontent.com/u/2250844?s=460&u=24deb32096e9f892cc91a6ff1ca1af50193b1fbd&v=4
+description: Frontend Asset Passthrough
+image: https://user-images.githubusercontent.com/4712430/104760766-7a2c5980-5727-11eb-93f5-3296b23ba3a0.png
+hide_table_of_contents: false
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Ever wonder how Clutch handles serving frontend assets without a CDN in a distributed multi-version deployment?
+This article will touch on how Lyft deploys Clutch and some of the early problems we faced when rolling it out.
+
+<!--truncate-->
+
+## Problem
+
+When deploying Clutch at Lyft we noticed very early on when deploying a new version,
+a subset of users would fail to load the webpage as some of the frontend assets could not be resolved.
```

Comment:
```suggestion
a subset of users would fail to load the webpage (only seeing a wall of white) as some of the frontend assets could not be resolved.
```

Reply:
it would be awesome if we could find the JS/browser console error message that occurs

Reply:
agreed. Looking through our chat the console error was
`Uncaught Syntax Error: Unexpected token '<'`. I can send you a screenshot in a DM if you want to include that.

Reply:
Yeah nice let me add that